src/layout/Layout.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import Menu from '../components/Menu';
+import MenuItem from '../components/MenuItem';
+import routes from '../routes';
+import { Route, Switch } from 'react-router-dom';
+
+const Layout = props => {
+  const [active, setActive] = useState({});
+  const src = "https://i.pravatar.cc/75?img=2";
+  
+  function isActive(href) {
+    return window.location.pathname === href;
+  }
+
+  const click = () => {
+    setActive({active: null});
```

Comment:
```suggestion
    setActive({});
```
да можно и просто новый объект передавать